mixer/adapter/opa/opa_integration_test.go

```diff
@@ -242,18 +241,18 @@ func xTestServer(t *testing.T) {
 				"destination.service": "svc.cluster.local",
 				"source.uid":          "ken",
 				"request.path":        "/reviews/janet",
-				"target.service":      "details",
-				"source.service":      "landing_page",
+				"target.service":      "landing_page",
+				"source.service":      "details",
 			},
 			expectedStatusCode: 0,
 		},
 		"Denied request": {
 			attrs: map[string]interface{}{
 				"destination.service": "svc.cluster.local",
 				"source.uid":          "janet",
-				"request.path":        "/detail/janet",
-				"target.service":      "invalid",
-				"source.service":      "landing_page",
+				"request.path":        "/detail/ken",
+				"target.service":      "landing_pages",
```

Comment:
i don't want to block this PR, but what is the difference between `target.service` and `destination.service` and how do these relate to attributes?

Reply:
destination.service is not used in the test policy. It can be removed.

Reply:
I was wrong. destination.service  is required.
target service is used to fill input.action.service in the test configuration.